server/server.js

```diff
@@ -0,0 +1,26 @@
+const express = require('express');
+const app = express();
+const cors = require('cors');
+const knex = require('./knex');
+const port = 3001;
+
+app.set('port', port);
+
+app.use(cors());
+app.use(express.json());
+
```

Comment:
You might need to add one more method here to be able to parse the information coming in with post request.